### ISSUE TITLE
[BUG] List event handlers and accessibility issues/requests

### ISSUE THREAD

**Issue by mmmtrifork** [kirbydesign/designsystem#256]
**Describe the bug**
1. We are able to iterate through list items (by pressing 'tab' on the keyboard), but we are not able to select any of them (by pressing 'space' or 'enter')
2. Hovering on selectable list items does not change the cursor to a pointer

**To Reproduce**

1. Go to https://cookbook.kirby.design/home/showcase/action-sheet
2. Click on 'Show action sheet'
3. 'Tab' select any of the items
4. Press space or enter - nothing happens
5. Hover it with your mouse - it does not turn into a pointer 

**Kirby version**
 - [0.0.136]

**Expected behavior**
List entries should be operated by mouse, touch and keyboard users. This implies that pressing either 'space' or 'enter' should trigger a 'click' event on a list item, just as <button> elements work. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Required_JavaScript_Features for more details. Additionally, hovering selectable list items should transform the mouse cursor into a pointer.  

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Chrome
 - Version Newest

**Comment by jakobe** [comment]
We should replace our own web list implementation with Ionic'ls list component:
https://ionicframework.com/docs/api/list

**Comment by hafstad** [comment]
#451
